subsys/bluetooth/controller/util/util.c

```diff
@@ -55,3 +59,175 @@ int util_rand(void *buf, size_t len)
 
 	return entropy_get_entropy(dev, (uint8_t *)buf, len);
 }
+
+/** @brief Prepare access address as per BT Spec.
+ *
+ * - It shall have no more than six consecutive zeros or ones.
+ * - It shall not be the advertising channel packets' Access Address.
+ * - It shall not be a sequence that differs from the advertising channel
+ *   packets Access Address by only one bit.
+ * - It shall not have all four octets equal.
+ * - It shall have no more than 24 transitions.
+ * - It shall have a minimum of two transitions in the most significant six
+ *   bits.
+ *
+ * LE Coded PHY requirements:
+ * - It shall have at least three ones in the least significant 8 bits.
+ * - It shall have no more than eleven transitions in the least significant 16
+ *   bits.
+ */
+int util_aa_le32(uint8_t *dst)
```

Comment:
It is not obvious that aa stands for access address, propose to write this out fully in a later update